src/call-hierarchy.ts

```diff
@@ -210,11 +228,21 @@ export class CallHierarchy {
       return undefined;
     }
   };
-  public getOutgoingCallHierarchy = ():
-    | CallHierarchyItemWithChildren
-    | undefined => {
-    const item = this.prepareCallHierarchy();
+  public getOutgoingCallHierarchy = (
+    callSite: CallSite,
+  ): CallHierarchyItemWithChildren | undefined => {
+    const item = this.prepareCallHierarchy(callSite);
     if (!item) return undefined;
-    return this.getOutgoingCalls(item);
+    const ch = this.getOutgoingCalls(item);
+    if (!ch) return undefined;
+    return {
+      id: 0, // The original call site starts with index 0.
+      name: callSite.calledFunction,
+      kind: ts.ScriptElementKind.functionElement,
+      file: callSite.fileName,
+      selectionRange: callSite.realPosition,
+      range: callSite.realPosition,
+      children: [ch],
+    };
```

Comment:
original call site + the following callees are necessary combo.
So just unify them here instead of spliting.

Reply:
id starts with index 0